### ISSUE TITLE
Can't ls binary key names

### ISSUE THREAD

**Issue by elisaado** [muesli/thunder#6]
Hi, when I'm trying to `ls` I get

```
[blog.db /User/__storm_index_ID] # ls
25 keys in bucket (25 omitted in this list)
[blog.db /User/__storm_index_ID] #  
```

**Comment by muesli** [comment]
This means that the keys' names in this bucket aren't displayable, e.g. because they are binary values / encoded. I'm still looking for good ideas how to handle such situations. For example we could assign names like 'binarykey_1'.

**Comment by elisaado** [comment]
Oh, I see. A good idea might be an option to ls (e.g. `ls -a`) to force display it but converted to a string. Assigning names is probably the best idea.

Should I keep this issue open? I thought about closing it when we have some kind of way to display them.

**Comment by muesli** [comment]
Yeah, keep it open, please. We should definitely improve the situation.
